src/controller/session/authUser.ts

```diff
@@ -0,0 +1,34 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { InMemoryUserRepository } from '../../repositories/in-memory-db/inMemoryUserRepository'
+import { AuthUserUseCase } from '../../use-cases/authUserUseCase'
+import { z } from 'zod'
+
+export async function authUser( 
+  request: FastifyRequest,
+  response: FastifyReply,
+) {
+
+  const userRepository = new InMemoryUserRepository()
```

Comment:
We should use the Prisma repository as it's going to production 

This is the file (master branch)
https://github.com/MatheusSanchez/orange-back/blob/master/src/repositories/prisma/prisma-users-repository.ts